vcluster/deploy/control-plane/binary/README.mdx

```diff
@@ -14,6 +14,20 @@ import TenancySupport from '../../../_fragments/tenancy-support.mdx';
 
 <Standalone />
 
+## Hardware requirements
+The recommended resource configuration for vCluster Standalone is as follows:
+```yaml title="Resource requirements"
```

Comment:
These requirements are for server / VM where you install vCluster Standalone, it is not something that you configure in the `vcluster.yaml` or elsewhere.
So I suggest rephrasing it to something like:

> The minimal recommended resource requirements for vCluster Standalone are as follows:
> - 1 vCPU
> - 2 GB of RAM